### PR TITLE
Added tests for Numba ScalarOps

### PR DESCRIPTION
**Pull request by kc611**
This PR should cover most of the `ScalarOps` from `basic.py`

### CODE REVIEW

File: tests/link/test_numba.py [hunk 1]
```diff
@@ -86,6 +115,67 @@ def test_Elemwise(inputs, input_vals, output_fn):
     compare_numba_and_py(out_fg, input_vals)
 
 
+@pytest.mark.parametrize(
+    "unary_op, x_range",
+    [
+        (inv, list(range(1, 10))),
```
Comment (kc611) [inline tests/link/test_numba.py:121]: The tests in `binary.py` check for a much larger interval. But that would make these tests a bit slow. So shortened the interval as we only need to see the functionality of the Numba `Op`.

Comment (michaelosthege) [reply tests/link/test_numba.py:121]: Or you can use `np.linspace` on a large interval, with a specified number of points? Then they also take float values inbetween.

**Review by michaelosthege — DISMISSED** [review]

**Review by brandonwillard — CHANGES_REQUESTED** [review]
These intervals look like they could be overkill.

Always start with this question: what **exactly** are you testing?  Remember, we are **not** testing Numba's/NumPy's/SciPy's implementation of these scalar functions.

**Comment by kc611** [comment]
>These intervals look like they could be overkill.

Should I change them to singular integer values rather than an interval ?

> Also, take a look at the coverage..

What are the changes that I should make to improve that ?

**Comment by michaelosthege** [comment]
Suggestions:
* split the list of Ops by their support and then just iterate them in a two tests, instead of creating a lot of tests with parametrize
* test for things that fail, not only things that work (what would that be?)
* The coverage did not increase, so either your tests test something that was already covered (or at least executed in other tests) - it does not mean that the other tests asserted something.. Or the code you are testing falls out of the coverage calculation because it is compiled

**Comment by kc611** [comment]
Sorry for late response. 

> The coverage did not increase, so either your tests test something that was already covered

I think that's probably because `ScalarOp`s like `Add` and `Mul` are being used in `test_Elemwise`. My intention with this particular test is to check the `Numba` conversion of each `ScalarOp`.

**Comment by brandonwillard** [comment]
This PR looks like it's been overshadowed by more recent Numba developments, so I'm going to close it for now.

**Pull request closed without merging**